### ISSUE TITLE
RSS and Sitemap Support

### ISSUE THREAD

**Issue by ccorda** [patronage/bubs-next#16]
Can we proxy WP's rss/xml?

**Comment by ccorda** [comment]
Here is a WP example:
https://dev.to/kendalmintcode/create-a-next-js-rss-feed-for-your-static-website-210p

Uses graphql to grab the necessary pieces, and outputs.

**Comment by ccorda** [comment]
@kylehotchkiss we tried the proxy approach recently, do you think that's viable here?

**Comment by kylehotchkiss** [comment]
I proposed a solution in another project (didn't actually code it out) but I had bubs in mind while brainstorming and think the Proxy+Rewrite solution is very viable. It's not too far off from how we implemented Hybrid sites before which are proven in production.

> Using Next.js' Rewrites functionality, any request to a file that matches a regex like /sitemap(.*).xml should rewrite to a dedicated Next.js page pages/sitemap.xml.js.

Implementation-wise I'm still in favor of rewriting to an /api/ over a /pages file, since outputting non-HTML from /pages files is hacky and might break in future next.js updates. 

Want me to grab this one?
